### ISSUE TITLE
How to implement ignoring label in training step?

### ISSUE THREAD

**Issue by barrykui** [facebook/fb.resnet.torch#146]
Does anyone how to ignore a specified label in the training and testing step? It means the ignored label will not have any contribution to the training loss and the accuracy, just like caffe set ignore label as 255.

**Comment by colesbury** [comment]
The easiest thing would probably to skip those samples in the data loader
